component-samples/demo/README.MD

```diff
@@ -209,25 +230,88 @@ The FDO PRI HTTP Java Device Sample currently supports `fdo_sys` and `fidoallian
 
     *exec-cb* - This command returns the status / progress of the process running.
 
+    Sample SVI instruction :
+  
+    `{"filedesc" : "sample.txt","resource" : "https://www.google.com"},
+     {"write": "sample.txt"},
+     {"exec" :"/bin/bash cat sample.txt"}`
 
 ***NOTE***: The comma-separated values must be ordered such that the 'filedesc' and 'write' objects are one after the other pair-wise, followed by the 'exec' commands.
 
 - `devmod` Device ServiceInfo module: This module supports multiple messages as listed down in the protocol specification, that is sent to the Owner as Device ServiceInfo. A sample format looks like 'devmod:active=1'.
 
 - Ensure that before each module the `module_name:active` message is sent as `true` for a successful ServiceInfo transfer. The device will ignore all ServiceInfo module messages if module_name:active doesn't precede the messages of the module.
 
-The FDO PRI Owner Sample currently supports the same `fdo_sys` module to send Owner ServiceInfo to the Device and `devmod` module to store the received Device ServiceInfo. Few sample ServiceInfo values, as present in owner/serviceinfo/sample-values/ are populated into the database table 'OWNER_SERVICEINFO' as byte arrays. For keeping the association between the Device and the ServiceInfo values to transfer, 'GUID_OWNERSVI' database table is used. When a Device is inserted into the database table 'TO2_DEVICES', it'll not have any association with the ServiceInfo values, and so by default, no ServiceInfo is transferred to the Device.
+The FDO PRI Owner Sample currently supports the same `fdo_sys` module to send Owner ServiceInfo to the Device and `devmod` module to store the received Device ServiceInfo. When a Device is inserted into the database table 'ONBOARDING_CONFIG', it'll not have any association with the ServiceInfo values, and so by default, no ServiceInfo is transferred to the Device.
```

Comment:
what does "Same" fdo_sys signify?

Reply:
Typo. Updated:
The FDO PRI Owner Sample currently supports `fdo_sys` module for sending ServiceInfo to the PRI Device and `devmod` module to store the received Device ServiceInfo. When a Device is inserted into the database table 'ONBOARDING_CONFIG', it'll not have any association with the ServiceInfo values, and so by default, no ServiceInfo is transferred to the Device.